composer.json

```diff
@@ -43,20 +42,20 @@
     },
     "autoload": {
         "psr-4": {
-            "Liip\\FunctionalTestBundle\\": "src/"
+            "Liip\\TestFixturesBundle\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Liip\\FunctionalTestBundle\\Tests\\": "tests/"
+            "Liip\\Acme\\Tests\\": "tests/"
```

Comment:
Good move! :+1:

Reply:
Thanks, dealing with `Liip\\FunctionalTestBundle\\Tests\\` and `Liip\\FunctionalTestBundle\\Test\\` was confusing. :)